contrib/aws-tag-lb-service-webhook/main.go

```diff
@@ -21,18 +24,21 @@ func mutateService(w http.ResponseWriter, r *http.Request) {
 
 	data, err := io.ReadAll(r.Body)
 	if err != nil {
+		log.Printf("failed to read request body: %v\n", err)
 		http.Error(w, "Failed to read request body", http.StatusBadRequest)
 		return
 	}
 
 	if err := json.Unmarshal(data, &admissionReview); err != nil {
+		log.Printf("failed to unmarshal admission review: %v\n", err)
```

Comment:
You're going to want to see what the error is even if you don't send it to the user in the response, ask me how I know ...